### PR TITLE
Update examples for the converged Switch component

### PR DESCRIPTION
**Pull request by andybb**
This will add the examples to the Switch component on the website.

### CODE REVIEW

File: Examples/UIExplorer/SwitchExample.js [hunk 1]
```diff
@@ -138,15 +135,23 @@ exports.examples = [
     render(): ReactElement { return <DisabledSwitchExample />; }
   },
   {
-    title: 'Custom colors can be provided',
-    render(): ReactElement { return <ColorSwitchExample />; }
-  },
-  {
     title: 'Change events can be detected',
```
Comment (ide) [inline Examples/UIExplorer/SwitchExample.js]: Did you mean to delete this example?

Comment (andybb) [inline Examples/UIExplorer/SwitchExample.js]: ColorSwitchExample is only added on iOS. See the code a bit further down.

**Comment by satya164** [comment]
@facebook-github-bot shipit

**Comment by bestander** [comment]
Import failed becuase out SnapshotTests in closed source depend in SwitchIOSExample.

**Comment by bestander** [comment]
I'll do it manually

**Comment by bestander** [comment]
@andybb could you please rebase?
File is conflicting.

**Comment by andybb** [comment]
@bestander Done.

**Comment by bestander** [comment]
@facebook-github-bot shipit

**Comment by bestander** [comment]
@facebook-github-bot shipit

**Comment by bestander** [comment]
One of the closed source snapshot tests was failing because the order of switches changed in the Example file.
I'll update the test and it should land within an hour.

**Comment by bestander** [comment]
closed with commit https://github.com/facebook/react-native/commit/05f31a0cbfaad33d8f67e2932f29bfc05556968a

**Pull request closed without merging**